platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationImplTest.java

```diff
@@ -108,10 +108,21 @@ public void testMissingVersionInName() throws Exception {
     @Test(expected = Exception.class)
     public void testUnparsableApplicationName() throws IOException {
         Repository repo = mock(Repository.class);
-        when(repo.getName()).thenReturn("MyName-0.0.0-1.1");
```

Comment:
the test is UnparsableApplicationName and it started failing because it is parseable after my change so I fixed it to fail again.